### DISCUSSION TITLE
2.4.40 Data time in alerts and hunt

### DISCUSSION THREAD

**Discussion by gbrodt** [Security-Onion-Solutions/securityonion#12503] [General]
Is there a way to stop the date/time range from changing when you navigate to another item and then come back.  When I set a range to look at in alerts and then go to hunt it changes the window of time. When I go back to alerts it changes again. When I put a date/time range in I would like it to stay unless I actually readjust it.  Seems to do same thing with dashboards and cases too.

**Comment by InfosecGoon** [comment]
The time range is stored on the browser side, and it's separate variables for the time ranges in the different subsystems (Cases, Hunt, Dashboards, Alerts). If you'd like to change the default value, you can do that in Administration --> Configuration --> SOC --> Config --> Server --> [module] --> relativeTimeUnit/relativeTimeValue.
